### PR TITLE
separate analytics tracker for voice wave avatar

### PR DESCRIPTION
**Pull request by mritunjayz**

### CODE REVIEW

File: web/src/components/pages/dashboard/stats/leaderboard-card.tsx [hunk 1]
```diff
@@ -127,6 +127,9 @@ class UnconnectedLeaderboard extends React.Component<Props, State> {
   }
 
   playAvatarClip = function(clipUrl: string, position: any) {
+    const { locale } = this.props;
+    trackVoiceAvatar('listen', locale);
```
Comment (Gregoor) [inline web/src/components/pages/dashboard/stats/leaderboard-card.tsx]: a bit of an edge-casy nit: could we trigger self-listen here when a user clicks on their own row?

**Comment by Gregoor** [comment]
looks great overall! just a tiny nit :)

File: web/src/components/pages/dashboard/stats/leaderboard-card.tsx (hunk 1 above)
Comment (mritunjayz) [reply web/src/components/pages/dashboard/stats/leaderboard-card.tsx]: yeah! :+1:

**Comment by mritunjayz** [comment]
Done !

File: web/src/components/pages/dashboard/stats/leaderboard-card.tsx [hunk 2]
```diff
@@ -126,9 +126,10 @@ class UnconnectedLeaderboard extends React.Component<Props, State> {
     );
   }
 
-  playAvatarClip = function(clipUrl: string, position: any) {
+  playAvatarClip = function(clipUrl: string, position: any, self: boolean) {
     const { locale } = this.props;
-    trackVoiceAvatar('listen', locale);
+    if (self) trackVoiceAvatar('self-listen', locale);
+    else trackVoiceAvatar('listen', locale);
```
Comment (Gregoor) [inline web/src/components/pages/dashboard/stats/leaderboard-card.tsx]: personally I'd go [ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here, e.g. `trackVoiceAvatar(self ? 'self-listen' : 'listen', locale)`, but I'm happy to merge it in either way. Thanks! :)

**Pull request merged**